next/utils/base64.ts

```diff
@@ -0,0 +1,7 @@
+export const base64Encode = (input: string) => {
+  return Buffer.from(input).toString('base64')
+}
+
+export const base64Decode = (input: string) => {
+  return Buffer.from(input, 'base64').toString('utf8')
+}
```

Comment:
would mark this as backedn specific - FE does not have Buffer (I think ?), but has window.btoa and window.atob